tag-security-and-privacy.md

```diff
@@ -0,0 +1,75 @@
+# TAG Review: Security and Privacy questionnaire
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+There is an ongoing discussion ([tc39/proposal-import-attributes#61](https://github.com/tc39/proposal-import-attributes/issues/61)) about communicating the `type` attribute to the origin. For instance:
+
+```js
+import json from "/foo.json" with type: "json";
+```
+
+would presumably send `Accept: application/json` to the origin.
+
+## 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes.
+
+## 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+No extra personal information or personally-identifiable information is exposed by this proposal.
```

Comment:
Suggested wording: HTML imports modules by performing fetches from the URL indicated in the module specifier. JavaScript code may construct a URL exposing personally identifying information which is implicitly communicated by importing a particular kind of JSON module, but this is already possible with JS modules. No new identifying information is communicated.